lib/blocs/upload/upload_handles/bundle_upload_handle.dart

```diff
@@ -50,21 +51,37 @@ class BundleUploadHandle implements UploadHandle {
     required Wallet wallet,
   }) async {
     final bundle = await DataBundle.fromHandles(
+      parallelize: false,
```

Comment:
I'm not sure about this line. I'll see if I find out when reviewing Arweave-Dart

Reply:
If running this method synchronously is the fix and this affects ArConnect only, then we could do something like...
```suggestion
      parallelize: !isArConnect,
```

Reply:
That makes total sense. Thanks for this good catch!

Reply:
[fc40ea6](https://github.com/ardriveapp/ardrive-web/pull/742/commits/fc40ea60ee52bb7cccabbfd548f10234e43ae643)